{
  "uid" : "4edff89545dd49cd",
  "name" : "[US-111]/[2] I can't log into automationpractice.com using email \"thanos.ALars@example.com\" & password \"12345\"",
  "fullName" : "tests.WebAPP.Authentication_Tests.test_2",
  "historyId" : "9b3069251ad31cc0a7516a6160d05383",
  "time" : {
    "start" : 1602694631316,
    "stop" : 1602694637044,
    "duration" : 5728
  },
  "description" : "[US-111]/[2] As a user I can't log into automationpractice.com using email \"thanos.ALars@example.com\" & password \"12345\"",
  "descriptionHtml" : "<p>[US-111]/[2] As a user I can't log into automationpractice.com using email &quot;thanos.ALars@example.com&quot; &amp; password &quot;12345&quot;</p>\n",
  "status" : "failed",
  "statusMessage" : "Ups, something went really bad! Even Michael Scofield couldn't have predicted that error! :) expected [http://automationpractice.com/index.php] but found [http://automationpractice.com/]",
  "statusTrace" : "java.lang.AssertionError: Ups, something went really bad! Even Michael Scofield couldn't have predicted that error! :) expected [http://automationpractice.com/index.php] but found [http://automationpractice.com/]\n\tat org.testng.Assert.fail(Assert.java:99)\n\tat org.testng.Assert.failNotEquals(Assert.java:1037)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:140)\n\tat org.testng.Assert.assertEquals(Assert.java:122)\n\tat org.testng.Assert.assertEquals(Assert.java:629)\n\tat com.steps.MainPageSteps.iOpenHomePage(MainPageSteps.java:35)\n\tat tests.WebAPP.Authentication_Tests.test_2(Authentication_Tests.java:73)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:564)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)\n\tat org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)\n\tat java.base/java.lang.Thread.run(Thread.java:832)\n",
  "flaky" : false,
  "newFailed" : true,
  "beforeStages" : [ {
    "name" : "Setting up Excel File",
    "time" : {
      "start" : 1602694523548,
      "stop" : 1602694523599,
      "duration" : 51
    },
    "description" : "Setting up Excel File",
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "Setting up Test Class",
    "time" : {
      "start" : 1602694630373,
      "stop" : 1602694631312,
      "duration" : 939
    },
    "description" : "Setting up Test Class",
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "testStage" : {
    "description" : "[US-111]/[2] As a user I can't log into automationpractice.com using email \"thanos.ALars@example.com\" & password \"12345\"",
    "status" : "failed",
    "statusMessage" : "Ups, something went really bad! Even Michael Scofield couldn't have predicted that error! :) expected [http://automationpractice.com/index.php] but found [http://automationpractice.com/]",
    "statusTrace" : "java.lang.AssertionError: Ups, something went really bad! Even Michael Scofield couldn't have predicted that error! :) expected [http://automationpractice.com/index.php] but found [http://automationpractice.com/]\n\tat org.testng.Assert.fail(Assert.java:99)\n\tat org.testng.Assert.failNotEquals(Assert.java:1037)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:140)\n\tat org.testng.Assert.assertEquals(Assert.java:122)\n\tat org.testng.Assert.assertEquals(Assert.java:629)\n\tat com.steps.MainPageSteps.iOpenHomePage(MainPageSteps.java:35)\n\tat tests.WebAPP.Authentication_Tests.test_2(Authentication_Tests.java:73)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:564)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)\n\tat org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)\n\tat java.base/java.lang.Thread.run(Thread.java:832)\n",
    "steps" : [ {
      "name" : "I open home page",
      "time" : {
        "start" : 1602694636515,
        "stop" : 1602694636613,
        "duration" : 98
      },
      "status" : "failed",
      "statusMessage" : "Ups, something went really bad! Even Michael Scofield couldn't have predicted that error! :) expected [http://automationpractice.com/index.php] but found [http://automationpractice.com/]",
      "statusTrace" : "java.lang.AssertionError: Ups, something went really bad! Even Michael Scofield couldn't have predicted that error! :) expected [http://automationpractice.com/index.php] but found [http://automationpractice.com/]\n\tat org.testng.Assert.fail(Assert.java:99)\n\tat org.testng.Assert.failNotEquals(Assert.java:1037)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:140)\n\tat org.testng.Assert.assertEquals(Assert.java:122)\n\tat org.testng.Assert.assertEquals(Assert.java:629)\n\tat com.steps.MainPageSteps.iOpenHomePage(MainPageSteps.java:35)\n\tat tests.WebAPP.Authentication_Tests.test_2(Authentication_Tests.java:73)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:564)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)\n\tat org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)\n\tat java.base/java.lang.Thread.run(Thread.java:832)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ {
      "uid" : "8555133f3bec8640",
      "name" : "Scenario FAIL screenshot",
      "source" : "8555133f3bec8640.png",
      "type" : "image/png",
      "size" : 18343
    }, {
      "uid" : "3dd5a0a19edc01aa",
      "name" : "TestNG test FAIL logs",
      "source" : "3dd5a0a19edc01aa.txt",
      "type" : "text/plain",
      "size" : 2695
    } ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 2,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  },
  "afterStages" : [ {
    "name" : "Teardown Test Class",
    "time" : {
      "start" : 1602694637047,
      "stop" : 1602694637107,
      "duration" : 60
    },
    "description" : "Teardown Test Class",
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "Teardown Test Suite",
    "time" : {
      "start" : 1602695039259,
      "stop" : 1602695040517,
      "duration" : 1258
    },
    "description" : "Teardown Test Suite",
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "package",
    "value" : "tests.WebAPP.Authentication_Tests"
  }, {
    "name" : "testClass",
    "value" : "tests.WebAPP.Authentication_Tests"
  }, {
    "name" : "testMethod",
    "value" : "test_2"
  }, {
    "name" : "parentSuite",
    "value" : "Automationpractice.com WEB - Kamil Nowocin"
  }, {
    "name" : "suite",
    "value" : "Automated tests for WEB client"
  }, {
    "name" : "subSuite",
    "value" : "tests.WebAPP.Authentication_Tests"
  }, {
    "name" : "host",
    "value" : "travis-job-a8c8b075-3079-49fb-9f08-6d508dcf90e2"
  }, {
    "name" : "thread",
    "value" : "6396@travis-job-a8c8b075-3079-49fb-9f08-6d508dcf90e2.TestNG-test=Automated tests for WEB client-4(26)"
  }, {
    "name" : "framework",
    "value" : "testng"
  }, {
    "name" : "language",
    "value" : "java"
  }, {
    "name" : "story",
    "value" : "NEGATIVE FLOW"
  }, {
    "name" : "owner",
    "value" : "Kamil Nowocin"
  }, {
    "name" : "epic",
    "value" : "Web App Tests"
  }, {
    "name" : "feature",
    "value" : "LOGIN TESTS"
  }, {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ {
    "name" : "TAP-0002",
    "url" : "https://tracker.FAKE.com/jira/browse/AUTOMATION_PRACTICE/issue/TAP-0002",
    "type" : "issue"
  }, {
    "name" : "STORY-111",
    "url" : "https://tracker.FAKE.com/jira/browse/AUTOMATION_PRACTICE/STORY-111",
    "type" : "tms"
  } ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "owner" : "Kamil Nowocin",
    "retries" : [ {
      "uid" : "c194266bffc208b9",
      "status" : "skipped",
      "statusDetails" : "Expected condition failed: waiting for element to be clickable: Proxy element for: DefaultElementLocator 'By.xpath: //a[@class='login']' (tried for 15 second(s) with 500 milliseconds interval)",
      "time" : {
        "start" : 1602694604645,
        "stop" : 1602694630308,
        "duration" : 25663
      }
    } ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 2,
        "broken" : 2,
        "skipped" : 15,
        "passed" : 281,
        "unknown" : 0,
        "total" : 300
      },
      "items" : [ {
        "uid" : "84b689bdbd96d351",
        "reportUrl" : "https://travis-ci.com/kamil-nowocin/Test_Automation-automationpractice/builds/189766563/#testresult/84b689bdbd96d351",
        "status" : "passed",
        "time" : {
          "start" : 1602608223408,
          "stop" : 1602608238434,
          "duration" : 15026
        }
      }, {
        "uid" : "4b13417d49eda534",
        "reportUrl" : "https://travis-ci.com/kamil-nowocin/Test_Automation-automationpractice/builds/189536094/#testresult/4b13417d49eda534",
        "status" : "passed",
        "time" : {
          "start" : 1602521854342,
          "stop" : 1602521872261,
          "duration" : 17919
        }
      }, {
        "uid" : "bbb6a4f40e8feefc",
        "reportUrl" : "https://travis-ci.com/kamil-nowocin/Test_Automation-automationpractice/builds/189323165/#testresult/bbb6a4f40e8feefc",
        "status" : "passed",
        "time" : {
          "start" : 1602435370397,
          "stop" : 1602435386301,
          "duration" : 15904
        }
      }, {
        "uid" : "5b9fd5e89fc15094",
        "reportUrl" : "https://travis-ci.com/kamil-nowocin/Test_Automation-automationpractice/builds/189207237/#testresult/5b9fd5e89fc15094",
        "status" : "passed",
        "time" : {
          "start" : 1602348946521,
          "stop" : 1602348963819,
          "duration" : 17298
        }
      }, {
        "uid" : "f696bb67685b5640",
        "reportUrl" : "https://travis-ci.com/kamil-nowocin/Test_Automation-automationpractice/builds/189083485/#testresult/f696bb67685b5640",
        "status" : "passed",
        "time" : {
          "start" : 1602262516159,
          "stop" : 1602262533337,
          "duration" : 17178
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "4edff89545dd49cd.json",
  "parameterValues" : [ ]
}